katas/metal-slug/kevin/CharacterStatus.js

```diff
@@ -0,0 +1,28 @@
+export class CharacterStatus {
+    getLifePoints = (model, equipment) => {
+        const modelLifepoints = model;
+        const equipmentLifepoints = equipment;
+        let totalLifePoints = modelLifepoints + equipmentLifepoints;
+        return totalLifePoints;
+    };
+
+    getAttackPower = (model, equipment) => {
```

Comment:
Could you please elaborate what `model` and `equipment` variables mean? I see you have the same variables in `getLifePoints`, but the values seem to have a different meaning. For example, here you use equipment to define attack power, but in `getLifePoints` you use it to define life points, aren't we using the same variable value to set the value of two different things?